torch/csrc/stable/stableivalue_conversions.h

```diff
@@ -64,6 +64,9 @@ struct FromImpl {
     static_assert(
         !is_std_vector_v<T>,
         "std::vector<T> requires TORCH_FEATURE_VERSION >= TORCH_VERSION_2_10_0");
+    static_assert(
+        !std::is_same_v<T, std::string>,
```

Comment:
string_view get converted to string before this assert? Or we need to also assert about string_view here?

Reply:
so string_view doesn't have a .c_str(), so we do expect it to be converted to a std::string before getting called on by `from`.